### PR TITLE
fix: properly lex imports in OPL

### PR DESCRIPTION
**Pull request by hperl**
## Related issue(s)

Closes https://github.com/ory/keto/issues/1040

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: internal/schema/lexer.go [hunk 1]
```diff
@@ -307,7 +307,7 @@ func lexBlockComment(l *lexer) stateFn {
 func lexStringLiteral(l *lexer) stateFn {
 	r := l.next()
 	l.ignore()
-	l.acceptRun(digits + letters)
+	l.acceptRun(digits + letters + "@/-")
```
Comment (zepatrik) [inline internal/schema/lexer.go:310]: hm shouldn't we accept basically anything except unescaped `"` here anyways?

**Review by zepatrik — COMMENTED** [review]
So are `import`s now considered string literals, but their value is not used? Don't really understand how this works now :sweat_smile:

File: internal/schema/lexer.go (hunk 1 above)
Comment (hperl) [reply internal/schema/lexer.go:310]: Yes, we can do that.

**Comment by hperl** [comment]
> So are `import`s now considered string literals, but their value is not used? Don't really understand how this works now sweat_smile

In the lexing phase, we just want to generate a token stream from the input. So `import { Foo, Bar, Baz } from '@ory/keto-namespace-types'` becomes this:

```json
[
  "'import'",
  "{",
  "'Foo'",
  ",",
  "'Bar'",
  ",",
  "'Baz'",
  "}",
  "'from'",
  "'@ory/keto-...'",
  "EOF"
]
```

In the parsing phase, this stream will be complete ignored because we just skip until we find a `class` keyword.

**Review by zepatrik — APPROVED** [review]

**Pull request merged**